<launch>
    <node name="ply_filter" pkg="diplom" type="ply_filter_node" output="screen" required="true">

        <param name="input_ply_path" value="/home/prostagma/orb_maps/sdsadasdsadasdsadas.ply"/>

        <!-- <param name="output_ply_path" value="/home/prostagma/orb_maps/sdsadasdsadasdsadas_filtered.ply"/> -->

        <!-- Перезаписывать ли выходной файл, если он существует -->
        <param name="overwrite_output" value="true"/>

        <!-- Параметры для фильтра ROR -->
        <!-- Радиус поиска соседей -->
        <param name="ror_radius_search" value="0.5"/>
        <!-- Минимальное количество соседей в радиусе для сохранения точки -->
        <param name="ror_min_neighbors" value="3"/>
        <!-- Необязательный параметр: Размер листа воксельной сетки -->
        <!-- Если не указан или <= 0, будет равен ror_radius_search -->
        <param name="ror_voxel_leaf_size" value="0"/>

    </node>
</launch>